spring-cloud-task-batch/src/main/java/org/springframework/cloud/task/batch/partition/DeployerPartitionHandler.java

```diff
@@ -145,19 +141,23 @@
 
 	private boolean defaultArgsAsEnvironmentVars = false;
 
+	private ThreadPoolTaskExecutor threadPoolTaskExecutor;
```

Comment:
I would program to an interface here (like `TaskExecutor`) and not a specific implementation. The user can then choose which implementation to use to launch tasks. Do you agree?